app/scripts/stores/mainStore.js

```diff
@@ -328,10 +328,8 @@ export class MainStore {
             let headers = json[1].map;
             if(page <= 1) {
                 this.projects = results;
-                this.listItems = results;
             } else {
                 this.projects = [...this.projects, ...results];
-                this.listItems = [...this.listItems, ...results];
```

Comment:
Will this break ListItems in `{dds}/projects/{:id}` and `{dds}/folders/{:id}`

Reply:
No. listItems should never be set in getProjects. They are separate concerns. I think this is something you added at some point and forgot to get rid of. It's not in ua_test or prod.